server/src/internalClusterTest/java/org/opensearch/search/sort/FieldSortIT.java

```diff
@@ -197,7 +197,8 @@ public void testIssue8226() throws InterruptedException {
     public void testIssue6614() throws ExecutionException, InterruptedException {
         List<IndexRequestBuilder> builders = new ArrayList<>();
         boolean strictTimeBasedIndices = randomBoolean();
-        final int numIndices = randomIntBetween(2, 25); // at most 25 days in the month
+        // consider only 15 days of the month to avoid hitting open file limit
+        final int numIndices = randomIntBetween(2, 15);
```

Comment:
Thanks @Poojita-Raj for this quick fix. 
I have one question around how did we know that larger shard count is the underlying problem ? Does test fails consistently when using large number of indices such as 50 ?

Reply:
Yes, we saw reproducible failures with seeds (from previous gradle check failures) when we set it to 25 which stopped when value was set to 18 so to be on the safe side we are setting it to 15. ([Reference here](https://github.com/opensearch-project/OpenSearch/issues/11347#issuecomment-1934877127))